plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/schema/confluent/AvroSchemaConverter.java

```diff
@@ -58,7 +58,9 @@
 
 public class AvroSchemaConverter
 {
-    public static final String DUMMY_FIELD_NAME = "dummy";
+    public static final String DUMMY_FIELD_NAME = "$cdekvwhddlgixmua";
```

Comment:
Please do not use random hardcoded characters. I liked `$dummy` more. Maybe `$empty_field_marker`?

Reply:
What concerns do you have?

Reply:
Should field have random part and $dummy part, e.g. `cdekvwhddlgixmua$dummy` ?

Reply:
It leaks to users. It is even documented. So if it is documented it cannot be  so cryptic, and it has to be more user (human) friendly.

Reply:
Sounds good, will update, thanks!

Reply:
I went with `$empty_field_marker` - seems more descriptive than `$dummy`, lmk.